x-pack/plugins/security_solution/public/timelines/components/timeline/styles.tsx

```diff
@@ -29,7 +29,7 @@ export const TimelineBody = styled.div.attrs(({ className = '' }) => ({
   overflow: auto;
   scrollbar-width: thin;
   flex: 1;
-  visibility: ${({ visible }) => (visible ? 'visible' : 'hidden')};
+  display: ${({ visible }) => (visible ? 'block' : 'none')};
```

Comment:
If we use `'visibility'` here, it still occupies space. The parent is 'flex' and Resolver is a sibling of this component. We need Resolver to grow. Seems like `display: none` did the trick. Anyone know of any reason why this would be bad?

@XavierM seems to know many things